src/3rd_party/CMakeLists.txt

```diff
@@ -5,11 +5,19 @@ add_subdirectory(./yaml-cpp)
 add_subdirectory(./SQLiteCpp)
 add_subdirectory(./pathie-cpp)
 add_subdirectory(./zlib)
-add_subdirectory(./faiss)
-include_directories(./faiss)
+#add_subdirectory(./faiss)
+#include_directories(./faiss)
 
-set(INTGEMM_DONT_BUILD_TESTS ON CACHE BOOL "Disable intgemm tests")
-add_subdirectory(./intgemm)
+
+if(COMPILE_CPU)
+  set(DNNL_BUILD_EXAMPLES OFF CACHE BOOL "Disable dnnl examples")
+  set(DNNL_BUILD_TESTS OFF CACHE BOOL "Disable dnnl tests")
+  add_subdirectory(./oneDNN)
+  include_directories(./oneDNN/include)
+
+  set(INTGEMM_DONT_BUILD_TESTS ON CACHE BOOL "Disable intgemm tests")
+  add_subdirectory(./intgemm)
```

Comment:
@XapaJIaMnu And I can remove `add_subdirectory(./intgemm)` this line as well. Right?

Reply:
Keep intgemm in, as it compiles fine and it might be useful down the line.